### ISSUE TITLE
Oneplus 7 Pro Android 11 Does Not Allways Log Background Location Usage

### ISSUE THREAD

**Issue by deivpaukst** [RushikeshKamewar/PrivacyDashboard#13]
The app works perfectly, the only issue is that it only logs location usage sometimes. The only thing I can think of that might be the cause is that the app can only have foreground location access.

**Comment by metronightly** [comment]
Yeah. Also the same can be said of microphone usage. The app doesn't log mic access by Google Assistant when the 'Hey Google any time screen is on' feature is on. It only logs when we open GA from the launcher or some other button.
